superchain/configs/sepolia/superchain.toml

```diff
@@ -8,6 +8,8 @@ ecotone_time = 1708534800 # Wed 21 Feb 2024 17:00:00 UTC
 fjord_time =   1716998400 # Wed 29 May 2024 16:00:00 UTC
 granite_time = 1723478400 # Mon Aug 12 16:00:00 UTC
 
+op_contracts_release = "op-contracts/v1.4.0"
```

Comment:
Nit: IMO this information also lives in the `validation/standard` package.

Reply:
I was going by the suggestion in https://github.com/ethereum-optimism/superchain-registry/pull/525#discussion_r1736585302. I've now moved the `op_contracts_release` to `validation/standard`. There is a slight change in the toml spelling due to this.

Because `standard-versions.toml` now has multiple elements, i've had to:
* wrap the map and the tag it in a struct on the go side.
* prefix the contracts release string with an explicit name.

The automatic unmarshaling doesn't work with just the string as the key to the map. Wanted to check what you think about the new spelling:

`standard-versions.toml`

```
op_contracts_release = "op-contracts/v1.4.0"

[contract_tag]

# Fault Proofs https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.4.0
[contract_tag."op-contracts/v1.4.0"]
address_manager = { version = "" }
l1_cross_domain_messenger = { version = "2.3.0" }
l1_erc721_bridge = { version = "2.1.0" }
...
# Fault Proofs https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.3.0
[contract_tag."op-contracts/v1.3.0"]
address_manager = { version = "1.0.0", address = "0xdE1FCfB0851916CA5101820A69b13a4E276bd81F" }
...
```

Reply:
I think that TOML layout is OK, it is still fairly legible. See other comment about whether we need to have multiple contract tags in this file, though.

Reply:
resolved per discussion in other comment